curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/set-the-childs-prototype-to-an-instance-of-the-parent.english.md

```diff
@@ -58,11 +58,11 @@ Animal.prototype = {
 
 function Dog() { }
 
-// Add your code below this line
+// Only change code below this line
 
 
 let beagle = new Dog();
-beagle.eat();  // Should print "nom nom nom"
```

Comment:
Same for this

Reply:
That is the point.  The user can write these and we do not have to translate the comment.  At this point in the curriculum, the user has already learned how to use `console.log`, so this would encourage debugging code.